config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: controller
+  newName: quay.io/rabi/keystone-operator
```

Comment:
This should be `quay.io/openstack-k8s-operators/keystone-operator`, or at least shouldn't be your own repo for now